src/js/dispatch/Dispatcher.js

```diff
@@ -0,0 +1,6 @@
+import { Dispatcher } from 'flux';
+
+/**
+ *  Send data to  all the registered ArticleStore and SourceStore
+ */
+export default new Dispatcher;
```

Comment:
I believe what you are doing here is export a new instance of Dispatcher. What you should have is...
```
export default new Dispatcher();
```